### PR TITLE
Added instructions to README on how to use paperclip with Simple Form

### PR DESCRIPTION
**Pull request by ahmad-alkheat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,13 @@ end
 <% end %>
 ```
 
+### Edit and New Views with Simple Form
+```erb
+<%= simple_form_for @user, :url => users_path, :html => { :multipart => true } do |form| %>
+   <%= form.input :avatar, as: :file %>
```
Comment (jyurek) [inline README.md]: Do we actually need the `multipart`? That should be handled for us if we had a `file` input.

Comment (ahmad-alkheat) [inline README.md]: You are right, I removed that part

**Comment by maclover7** [comment]
Hi @ahmad-alkheat ! Is this still an issue? If so, can you please rebase your PR and make sure everything is passing? Thanks!

**Comment by ahmad-alkheat** [comment]
@maclover7 Just rebased my commits. Thanks

**Comment by maclover7** [comment]
Looks great to me, thanks for submitting!

@jferris @jyurek Ready to merge?

**Comment by maclover7** [comment]
Looks ready to go to me.

paging @jferris @jyurek @mike-burns

**Comment by mike-burns** [comment]
Thanks; merged (with some formatting changes) as cdbc05eb508ad18a1fff746d2c4fa58c177ee837.

**Pull request closed without merging**